src/amo/components/HomepageShelves/index.js

```diff
@@ -75,10 +77,13 @@ export const HomepageShelvesBase = (props: InternalProps): React.Node => {
           ? INSTALL_SOURCE_FEATURED_COLLECTION
           : INSTALL_SOURCE_FEATURED;
 
-      const count =
-        endpoint === HOMESHELVES_ENDPOINT_SEARCH_THEMES
-          ? LANDING_PAGE_THEME_COUNT
-          : LANDING_PAGE_EXTENSION_COUNT;
+      const themeBool =
```

Comment:
`hasThemes` is a bit more readable and consistent with our codebase than `themeBool`.